examples/examples_nodejs_test.go

```diff
@@ -685,3 +685,78 @@ func TestAccCNIAcrossUpdates(t *testing.T) {
 	t.Log("Ensuring that re-running `pulumi up` results in no changes and no spurious diffs")
 	pt.Up(optup.ExpectNoChanges())
 }
+
+func TestAccAuthenticationMode(t *testing.T) {
```

Comment:
Note: these tests won't run in CI. You'd need to manually add them to the test matrix in our workflow files (eg. https://github.com/pulumi/pulumi-eks/blob/161e7af6803b58fea43ffdc3c6f9ee85cf658134/.github/workflows/run-acceptance-tests.yml#L422-L444).